doc/release-notes/0.14/0.14.md

```diff
@@ -88,6 +88,17 @@ The following table covers notable changes in v0.14. Further information about t
 <td valign="top">In earlier releases, DDR support was not available on the macOS platform, which affected problem diagnosis for the VM, garbage collector, and JIT compiler.</td>
 </tr>
 
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/4858">#4858</a></td>
+<td valign="top">New Java stack (jstack) tool</td>
+<td valign="top">OpenJDK8 and later (All platforms)</td>
+<td valign="top">Ability to obtain Java stack traces and thread information for processes.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/4655">#4655</a></td>
```

Comment:
This should be #4650, although #4655 does lead to it.

Reply:
Although everything else is an issue and not a PR. I was going to change it, but I think I'll just leave it be.